src/backend/distributed/transaction/worker_transaction.c

```diff
@@ -268,6 +265,30 @@ IsWorkerTransactionActive(void)
 
 
 /*
+ * RemoveWorkerTransaction removes the transaction connection to the specified node from
+ * the transaction connection list.
+ */
+void
+RemoveWorkerTransaction(char *nodeName, int32 nodePort)
+{
+	TransactionConnection *transactionConnection =
+		GetWorkerTransaction(nodeName, nodePort);
+
+	/* transactionConnection = NULL if the worker transactions have not opened before */
+	if (transactionConnection != NULL)
+	{
+		PGconn *connection = transactionConnection->connection;
+
+		PQexec(connection, "ROLLBACK");
```

Comment:
On second thought, maybe just remove it and just close the connection, since that also triggers rollback in a hands-off manner.